src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
As per the project requirements, all components should be class based, including the App.js file. Kindly change this function component to a class based component to meet the requirements :wink: